conformance/lang/lang-library/lang.regexp/find.balt

```diff
@@ -0,0 +1,92 @@
+Test-Case: output
+Description: Test `find` function.
+
+function init() {
+    var regExpr1 = re `Here`;
+    string str1 = "MatchHereAndHere";
+    regexp:Span? res1 = regExpr1.find("NotMatching");
+    io:println(res1 is ()); // @output true
+
+    regexp:Span? res2 = regExpr1.find(str1);
+    io:println(res2 is regexp:Span); // @output true
+    regexp:Span res2span = <regexp:Span> res2;
+    io:println(res2span.startIndex); // @output 5
+    io:println(res2span.endIndex);  // @output 9
+
+    regexp:Span? res3 = regExpr1.find(str1, 10);
+    io:println(res3 is regexp:Span); // @output true
+    regexp:Span res3span = <regexp:Span> res3;
+    io:println(res3span.startIndex); // @output 12
+    io:println(res3span.endIndex); // @output 16
+
+    regexp:Span? res4 = regExpr1.find(str1, 7);
+    io:println(res4 is regexp:Span); // @output true
+    regexp:Span res4span = <regexp:Span> res4;
+    io:println(res4span.startIndex); // @output 12
+    io:println(res4span.endIndex); // @output 16
+}
+
```

Comment:
Shall we test usages of character classes (`[ [^] [ReCharSet] ]`), capturing groups (`([? ReFlagsOnOff :] ReDisjunction)`)  and regular expressions with different quantifiers as well?